haas/api.py

```diff
@@ -687,6 +687,18 @@ def switch_delete_port(switch, port):
     db.session.delete(port)
     db.session.commit()
 
+@rest_call('GET', '/switches')
+def list_switches():
```

Comment:
Hey Ron.

Could you add a `Schema` argument, similar to this:
```python
@rest_call('GET', '/projects', Schema({}))
def list_projects():
```

Now that we've merged #596, all calls should have a `Schema`.

Thx!